src/components/Tables/ResourceTable.vue

```diff
@@ -32,20 +32,20 @@ export default {
   computed: {
     tableColumns() {
       return [
-        { key: "name", label: "Name" },
-        { key: "color", label: "Color" },
-        { key: "quality", label: "Quality" },
-        { key: "quantityType", label: "Quantity Type" },
-        { key: "shape", label: "Shape" },
-        { key: "size", label: "Size" },
-        { key: "type", label: "Type" },
-        { key: "purity", label: "Purity" },
-        { key: "plating", label: "Plating" },
-        { key: "carat", label: "Carat" },
-        { key: "cut", label: "Cut" },
-        { key: "dimensionX", label: "dimensionX" },
-        { key: "dimensionY", label: "dimensionY" },
-        { key: "dimensionZ", label: "dimensionZ" },
+        { clazz: "name", label: "Name" },
+        { clazz: "color", label: "Color" },
+        { clazz: "quality", label: "Quality" },
+        { clazz: "quantityType", label: "Quantity Type" },
+        { clazz: "shape", label: "Shape" },
+        { clazz: "size", label: "Size" },
+        { clazz: "type", label: "Type" },
+        { clazz: "purity", label: "Purity" },
+        { clazz: "plating", label: "Plating" },
+        { clazz: "carat", label: "Carat" },
+        { clazz: "cut", label: "Cut" },
+        { clazz: "dimensionX", label: "dimensionX" },
+        { clazz: "dimensionY", label: "dimensionY" },
+        { clazz: "dimensionZ", label: "dimensionZ" },
```

Comment:
Here it should be key in stead of clazz as well. But the first row should be: `{ key: "clazz", label: "Resource Type" }` (in stead of `{ clazz: "name", label: "Name" }`)